protocol/src/lib.rs

```diff
@@ -266,6 +296,8 @@ pub enum Message {
      * this is reconciliation work and not actual IO.  The reconciliation work
      * uses a different work queue and each reconciliation job must finish on
      * all three downstairs before the next one can be sent.
+     *
+     * TODO(matt) is this comment still correct?
```

Comment:
Yeah, it is still true.  The live repair is different than the initial reconciliation process that happens when we
first start up, though both make use of the same mechanism under the covers.

Reply:
Thanks, updated the comment.  At some point, I may add a `ReconciliationId` to make this _also_ strongly typed, but one thing at a time.